### PR TITLE
feature/support sports venues

### PR DESCRIPTION
**Pull request by istyf**
also includes a move of the domain package in its own commit

### CODE REVIEW

File: internal/pkg/application/facilities/sportsvenues.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	"github.com/diwise/context-broker/pkg/datamodels/diwise"
+	"github.com/diwise/context-broker/pkg/ngsild/client"
+	ngsierrors "github.com/diwise/context-broker/pkg/ngsild/errors"
+	"github.com/diwise/context-broker/pkg/ngsild/types/entities"
+	. "github.com/diwise/context-broker/pkg/ngsild/types/entities/decorators"
+	"github.com/diwise/context-broker/pkg/ngsild/types/properties"
+	"github.com/diwise/integration-cip-sdl/internal/pkg/domain"
+)
+
+var ErrSportsVenueIsOfIgnoredType error = errors.New("sportsvenue is of non supported type")
```
Comment (shestyf) [inline internal/pkg/application/facilities/sportsvenues.go:22]: sportsvenue should be two words

File: internal/pkg/application/facilities/sportsvenues.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+		if field.ID == 78 {
+			sportsVenue.Description = string(field.Value[1 : len(field.Value)-1])
+		} else if field.ID == 151 {
+			url := string(field.Value[1 : len(field.Value)-1])
+			url = strings.ReplaceAll(url, "\\/", "/")
+			sportsVenue.SeeAlso = []string{url}
+		}
+	}
+
+	// TODO: Fix these
```
Comment (shestyf) [inline internal/pkg/application/facilities/sportsvenues.go:122]: fix it then?

File: internal/pkg/application/facilities/sportsvenues.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+)
+
+var ErrSportsVenueIsOfIgnoredType error = errors.New("sportsvenue is of non supported type")
+
+func StoreSportsVenuesFromSource(logger zerolog.Logger, ctxBrokerClient client.ContextBrokerClient, ctx context.Context, sourceURL string, featureCollection domain.FeatureCollection) error {
+
+	headers := map[string][]string{"Content-Type": {"application/ld+json"}}
+
+	isSupportedType := func(t string) bool {
+		return t == "Badhus" || t == "Ishall" || t == "Sporthall"
```
Comment (nordbergmikael) [inline internal/pkg/application/facilities/sportsvenues.go:29]: dropdown?

**Review by shestyf — CHANGES_REQUESTED** [review]
.

File: internal/pkg/application/facilities/sportsvenues.go (hunk 3 above)
Comment (istyf) [reply internal/pkg/application/facilities/sportsvenues.go:29]: yep

**Review by shestyf — APPROVED** [review]
/lgtm

**Pull request merged**